src/services/forecast.ts

```diff
@@ -25,16 +27,29 @@ export class Forecast {
   ) {}
 
   public async processForecastForBeaches(
-    beaches: Beach[]
+    beaches: Beach[],
+    {
+      customProp,
+      ordination,
+    }: {
+      customProp: keyof Omit<BeachForecast, '_id'>;
+      ordination: Ordination;
+    } = {
+      customProp: 'rating',
+      ordination: Ordination.DESC,
+    }
   ): Promise<TimeForecast[]> {
     try {
       const beachForecast = await this.calculateRating(beaches);
       const timeForecast = this.mapForecastByTime(beachForecast);
       return timeForecast.map((t) => ({
         time: t.time,
-        // TODO Allow ordering to be dynamic
-        // Sorts the beaches by its ratings
-        forecast: _.orderBy(t.forecast, ['rating'], ['desc']),
+        forecast: t.forecast.sort((curr, next) =>
```

Comment:
Can you move the sort logic to a new PR? it's easier to review and we can merge them separately.

- [ ] Move sort logic to a new PR

Reply:
ok, I removed the following logic to another PR.